students/uss_tomasz/lesson_05_lists/5_6/age_calculator.py

```diff
@@ -0,0 +1,19 @@
+# Problem Age calculator
+from random import randint
+
+
+def age_calculator(peoples_age_list):
+    """For given list of people's ages calculate
+    average age of adults (age >= 18)
+    count the children (age < 18)
+    """
+    adults_list = [i for i in peoples_age_list if i >= 18]
+    average_adult = sum(adults_list)/len(adults_list)
+    children_count = len([i for i in peoples_age_list if i < 18])
+    return average_adult, children_count
+
+
+group_of_people = [randint(1, 30) for i in range(10)]
```

Comment:
same as above

Reply:
Please double-check. List comprahensions are used in lines 10 and 12. Or your comment was related with other issue?

Reply:
It refers to unseeded random number generator.